### ISSUE TITLE
Focus isn't on the correct element when publish dialog is opened

### ISSUE THREAD

**Issue by kisantia** [microsoft/azuredatastudio#17869]
There was some rearranging and new elements added to the publish project dialog in https://github.com/microsoft/azuredatastudio/pull/17495. When the dialog is opened, the focus should be on the first focusable element at the top of the dialog, which is the radio buttons, not the profile text box.
![image](https://user-images.githubusercontent.com/31145923/145287334-19a54079-f565-4929-b96b-e2683b3aa34b.png)
